### PR TITLE
Chronograf etcd migrations over TLS

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2735

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/chronograf/v1.9/administration/migrate-to-high-availability.md [hunk 1]
```diff
@@ -11,21 +11,43 @@ menu:
...
+    ##### Provide etcd TLS credentials
+    If TLS is enabled on `etcd`, provide your TLS certificate credentials using
+    the following query parameters in your etcd URL:
+
+    - **cert**: Path to client certificate file or PEM file
+    - **key**: Path to client key file
+    - **ca**: Path to trusted CA certificates
+
+    ```sh
+    etcd://127.0.0.1:2379?cert=/tmp/client.crt&key=/tst/client.key&ca=/tst/ca.crt
```
Comment (pierwill) [inline content/chronograf/v1.9/administration/migrate-to-high-availability.md:49]: Do these slashes need to be escaped, I wonder? :thinking:

File: content/chronograf/v1.9/tools/chronograf-cli/_index.md [hunk 2]
```diff
@@ -127,12 +127,15 @@ chronograf [flags]
...
+| Flag                     | Description                                                                                                | Env. Variable           |
+| :----------------------- | :--------------------------------------------------------------------------------------------------------- | :---------------------- |
+| `-e`, `--etcd-endpoints` | etcd endpoint URL (include multiple flags for multiple endpoints)                                          | `$ETCD_ENDPOINTS`       |
+| `--etcd-username`        | etcd username                                                                                              | `$ETCD_USERNAME`        |
+| `--etcd-password`        | etcd password                                                                                              | `$ETCD_PASSWORD`        |
+| `--etcd-dial-timeout`    | Total time to wait before timing out while connecting to etcd endpoints (0 means no timeout, default: -1s) | `$ETCD_DIAL_TIMEOUT`    |
+| `--etcd-request-timeout` | Total time to wait before timing out the etcd view or update (0 means no timeout, default: -1s)            | `$ETCD_REQUEST_TIMEOUT` |
```
Comment (pierwill) [inline content/chronograf/v1.9/tools/chronograf-cli/_index.md:136]: What does `-1` mean?

**Review by pierwill — APPROVED** [review]

File: content/chronograf/v1.9/administration/migrate-to-high-availability.md (hunk 1 above)
Comment (sanderson) [reply content/chronograf/v1.9/administration/migrate-to-high-availability.md:49]: These shouldn't need to be, but it depends on the shell the user is using I guess.

File: content/chronograf/v1.9/administration/migrate-to-high-availability.md [hunk 3]
```diff
@@ -11,21 +11,43 @@ menu:
...
 Update external links to Chronograf dashboards to reflect new source IDs.
 {{% /note %}}
 
 1. Stop the Chronograf server by killing the `chronograf` process.
 2. To prevent data loss, we **strongly recommend** that you back up your Chronograf data store before migrating to a Chronograf cluster.
 3. [Install and start etcd](/chronograf/v1.9/administration/create-high-availability/#install-and-start-etcd).
-4. Run the following command, specifying the local BoltDB file and the `etcd` endpoint beginning with `etcd://`. (We recommend adding the prefix `bolt://` to an absolute path. To specify a relative path to the BoltDB file, the prefix cannot be used.)
+4. Run the following command, specifying the local BoltDB file and the `etcd` endpoint beginning with `etcd://`.
+   (We recommend adding the prefix `bolt://` to an absolute path.
+   To specify a relative path to the BoltDB file, the prefix cannot be used.)
```
Comment (noramullen1) [inline content/chronograf/v1.9/administration/migrate-to-high-availability.md:24]:
```suggestion
   Do not use the prefix to specify a relative path to the BoltDB file.)
```

File: content/chronograf/v1.9/tools/chronograf-cli/_index.md (hunk 2 above)
Comment (sanderson) [reply content/chronograf/v1.9/tools/chronograf-cli/_index.md:136]: Not sure and the source code doesn't say either. We can circle back on this when we have more time.

File: content/chronograf/v1.9/tools/chronograf-cli/_index.md [hunk 4]
```diff
@@ -127,12 +127,15 @@ chronograf [flags]
...
+| `--etcd-cert`            | Path to PEM encoded TLS public key certificate for use with TLS                                            | `$ETCD_CERTIFICATE`     |
+| `--etcd-key`             | Path to private key associated with given certificate for use with TLS                                     | `$ETCD_PRIVATE_KEY`     |
+| `--etcd-root-ca`         | Path to root CA certificate for TLS verification.                                                          | `$ETCD-ROOT-CA          |
```
Comment (noramullen1) [inline content/chronograf/v1.9/tools/chronograf-cli/_index.md:139]:
```suggestion
| `--etcd-root-ca`         | Path to root CA certificate for TLS verification                                                          | `$ETCD-ROOT-CA          |
```

**Review by noramullen1 — APPROVED** [review]

**Pull request merged**